dm/pkg/binlog/event/common.go

```diff
@@ -163,7 +163,11 @@ func GTIDIncrease(flavor string, gSet gmysql.GTIDSet) (gmysql.GTIDSet, error) {
 		mariaGTID := singleGTID.(*gmysql.MariadbGTID)
 		mariaGTID.SequenceNumber++
 		gtidSet := new(gmysql.MariadbGTIDSet)
-		gtidSet.Sets = map[uint32]*gmysql.MariadbGTID{mariaGTID.DomainID: mariaGTID}
+		gtidSet.Sets = map[uint32]map[uint32]*gmysql.MariadbGTID{
```

Comment:
this and below part is copied from https://github.com/pingcap/tiflow/pull/10753/files#diff-7d0132544845cb9d5d4fbef67bdd54ba574cc5664f806e49fe1530aefad8cfa2